src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -401,7 +406,7 @@ public enum EvalMetricType
             }
             if (!string.IsNullOrEmpty(metric))
                 res["metric"] = metric;
-            res["sigmoid"] = "0.5";
+            res["sigmoid"] = Sigmoid.ToString();
```

Comment:
>Sigmoid [](start = 29, length = 7)

must be greater than zero according to [https://github.com/Microsoft/LightGBM/blob/master/docs/Parameters.rst](https://github.com/Microsoft/LightGBM/blob/master/docs/Parameters.rst)